metadata-ingestion/src/datahub/ingestion/source/superset.py

```diff
@@ -82,6 +131,61 @@
     "box_plot": ChartTypeClass.BAR,
 }
 
+SUPERSET_FIELD_TYPE_MAPPINGS: Dict[
+    str,
+    Type[
+        Union[
+            ArrayType,
+            BytesType,
+            BooleanType,
+            NumberType,
+            RecordType,
+            StringType,
+            TimeType,
+            NullType,
+        ]
+    ],
+] = {
+    "BYTES": BytesType,
+    "BOOL": BooleanType,
+    "BOOLEAN": BooleanType,
+    "DOUBLE": NumberType,
+    "DOUBLE PRECISION": NumberType,
+    "DECIMAL": NumberType,
+    "NUMERIC": NumberType,
+    "BIGNUMERIC": NumberType,
+    "BIGDECIMAL": NumberType,
+    "FLOAT64": NumberType,
+    "INT": NumberType,
+    "INT64": NumberType,
+    "SMALLINT": NumberType,
+    "INTEGER": NumberType,
+    "BIGINT": NumberType,
+    "TINYINT": NumberType,
+    "BYTEINT": NumberType,
+    "STRING": StringType,
+    "TIME": TimeType,
+    "TIMESTAMP": TimeType,
+    "DATE": TimeType,
+    "DATETIME": TimeType,
+    "GEOGRAPHY": NullType,
+    "JSON": NullType,
+    "INTERVAL": NullType,
+    "ARRAY": ArrayType,
+    "STRUCT": RecordType,
+    "CHARACTER VARYING": StringType,
+    "CHARACTER": StringType,
+    "CHAR": StringType,
+    "TIMESTAMP WITHOUT TIME ZONE": TimeType,
+    "REAL": NumberType,
+    "VARCHAR": StringType,
+    "TIMESTAMPTZ": TimeType,
+    "GEOMETRY": NullType,
+    "HLLSKETCH": NullType,
+    "TIMETZ": TimeType,
+    "VARBYTE": StringType,
```

Comment:
how was this list generated?

we've got a bunch of similar string type -> datahub type mappings, and ideally we'd reuse one of those instead of creating another one. The dbt source has a mapping that might be a good candidate for reuse

Reply:
This is pretty similar to the redshift mapping. Is this a file I can reference? Can't seem to find the dbt source mapping